### PR TITLE
Category about view

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@peterp depends on #56 and #66, otherwise ready for review.

File: src/categories/CategoryAbout/__tests__/CategoryAbout-test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          goal: 'Ut enim ad minim veniam, quis nostrud exercitation',
+        })}
+        {...props}
+      />
+    );
+  }
+
+  it('should render', () => {
+    const el = render(createComponent());
+    expect(el).toMatchSnapshot();
```
Comment (justinvdm) [inline src/categories/CategoryAbout/__tests__/CategoryAbout-test.js]: screen shot 2016-08-25 at 2 26 39 pm

File: src/categories/CategoryAbout/__tests__/CategoryAbout-test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  });
+
+  it('should render a fallback if the category does not have an image', () => {
+    const el = render(createComponent({ image: null }));
+    expect(el).toMatchSnapshot();
```
Comment (justinvdm) [inline src/categories/CategoryAbout/__tests__/CategoryAbout-test.js]: screen shot 2016-08-25 at 2 27 23 pm

**Comment by peterp** [comment]
👍

**Pull request merged**